spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-sample/src/test/java/com/example/PubSubSampleApplicationIntegrationTests.java

```diff
@@ -220,7 +213,7 @@ public void testMultiPull() {
 				.collect(Collectors.toList());
 	}
 
-	private void createTopic(String topicName) {
```

Comment:
Similar to a previous PR -- private methods should stay private unless they actually need to be accessed from outside of this class.